### PR TITLE
[AUT-2884] updated admin consent portal

### PR DESCRIPTION
**Pull request by ksroda-ce**
JIRA task: https://cloudentity.atlassian.net/browse/AUT-2884

Admin consent portal updated according to the designs (https://www.figma.com/file/9aHEVlmYsCgOBNgsobPvUb/Open-Banking-Demo---2021-Q2?node-id=3189%3A34790).
There was no design for the revoke confirm dialog, so I can update it later.

Screenshot 2021-06-16 at 14 22 40
Screenshot 2021-06-16 at 14 22 53
Screenshot 2021-06-16 at 14 23 01
Screenshot 2021-06-16 at 14 24 13
Screenshot 2021-06-16 at 14 24 16
Screenshot 2021-06-16 at 14 24 22
Screenshot 2021-06-16 at 14 24 26
Screenshot 2021-06-16 at 14 24 29
Screenshot 2021-06-16 at 14 24 33
Screenshot 2021-06-16 at 14 24 39
Screenshot 2021-06-16 at 14 24 42

### CODE REVIEW

**Review by ikawalec — APPROVED** [review]

**Review by piotrek-janus — APPROVED** [review]

**Comment by zspreacker-cloud** [comment]
@ksroda-ce On the payment access data table, given that it is already in the context of an application, please replace the column for `creditor` with `consent ID`.

**Comment by zspreacker-cloud** [comment]
@ksroda-ce I made a few updates to the design and prototype:
 - For completely revoking third-party access, I added a `side panel` interaction pattern that is the same as revoking consents.
 - Slight updates to make the button styles consistent for `Revoke` and `Manage`
 - After revoking a Third-party app, the row should appear disabled and move to the bottom of the list
 
 Updated Prototype:
 https://www.figma.com/proto/9aHEVlmYsCgOBNgsobPvUb/Open-Banking-Demo-2021-Q2?page-id=3189%3A34790&node-id=3679%3A16179&viewport=-825%2C7257%2C0.4634911119937897&scaling=min-zoom

File: consent/admin-portal/web/app/src/components/AccountView/Drawers/AccountAccessDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  drawerData: ClientType["consents"][0];
+  setDrawerData: (data: string | null) => void;
+};
+
+function AccountAccessDrawer({ drawerData, setDrawerData }: Props) {
+  const classes = useStyles();
+
+  const permissionDates = {
+    Authorised:
+      getDate(drawerData?.account_access_consent?.CreationDateTime) || "N/A",
```
[the remark this answers is not in the document]
Comment (ksroda-ce) [reply consent/admin-portal/web/app/src/components/AccountView/Drawers/AccountAccessDrawer.tsx]: :+1: fixed

**Comment by ksroda-ce** [comment]
@zspreacker-cloud ui updated

**Pull request merged**